README.md

```diff
@@ -2,51 +2,153 @@
 
 INBO protocols for monitoring
 
-*experimental*: we are in the process of trying out git-version control for INBO protocols. In the meantime, the official versions of INBO protocols are hosted [here](https://sites.google.com/a/inbo.be/veldprotocols/) (only accessible to INBO collaborators). Developmental versions are at [this google drive folder](https://drive.google.com/drive/folders/0BzUqT1wpznBXY2ZqaXh2a0tyd2M) (only accessible to INBO collaborators). 
+*experimental*: we are in the process of trying out git-version control for INBO protocols. In the meantime, the official versions of INBO protocols are hosted [here](https://sites.google.com/a/inbo.be/veldprotocols/) (only accessible to INBO collaborators). 
+Developmental versions are at [this google drive folder](https://drive.google.com/drive/folders/0BzUqT1wpznBXY2ZqaXh2a0tyd2M) (only accessible to INBO collaborators). 
 
 ## Repository structure
 
 ```
   |- README.md <- up to date description of the repo content
-  |- docs <- contains rendered versions of protocols
+  |- docs <- can be used to locally inspect rendered versions of protocols, gitignored
   |- from_docx <- contains Microsoft Word docx protocols which were converted to 
                   markdown
```

Comment:
I prefer to keep it as is. The docx files should not be git-ignored for archival reasons, availability of the file to both repo-admin and subject-matter specialists, and for checking if something went wrong with conversion.

The `docs` folder on the other hand, only serves for locally checking rendered result.

Reply:
> The docx files should not be git-ignored for archival reasons

I understand. Just an idea: when such a protocol has evolved to a newer version, IMO it would be good (and more consistent) to delete the older docx file. It will still be archived in the git repo (older commits). This would assist in providing just one version of each protocol in each release.

Reply:
e82afc6